src/main.rs

```diff
@@ -5,10 +5,11 @@ extern crate rocket;
 extern crate rocket_contrib;
 #[macro_use] extern crate serde_derive;
 
-extern crate pdf;
-
 use rocket_contrib::{Json};
-use pdf::pdf::Pdf;
+
+
+extern crate lib;
+use lib::entities::pdf::Pdf;
```

Comment:
I'm not really happy about this part. To have "entities" in "lib" dir.
Why not have two different dirs for these two? :-)

Reply:
Yeah, i don't really know what's supposed to go into the lib directory. So is it fine to have it like a normal project structure?

Reply:
In general, for this experimental project, structure is not so important :D

Reply:
But I'm fine to change crate name to lib or something similar :P

Reply:
Well, it is important, because it is an experimental project. So you know for the future how to do it. Yeah we could create more creates right? One lib, one entities and one util or something. Or put the util stuff into lib. I don't know :D